assets/helpers/abtests.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import type { Test } from './abtest';
+
+// ----- Tests ----- //
+
+export type Tests = { [testId: string]: Test }
```

Comment:
Since all the types are defined in `abtest.js` I would move this definition to that file, so everything is in one place and the functions (which receive `Tests`) are more readable.